test/unit/python/sample_files/write_init_files/physics_inputs_ddt2.F90

```diff
@@ -143,7 +143,7 @@ subroutine physics_read_data(file, suite_names, timestep, read_initialized_varia
 
                   select case (trim(phys_var_stdnames(name_idx)))
                      case ('potential_temperature')
-                        call read_field(file, 'potential_temperature', input_var_names(:,name_idx), 'lev', timestep, phys_state%theta)
+                        call read_field(file, 'potential_temperature', input_var_names(:,name_idx), 'lev', timestep, phys_state%thermo%theta)
```

Comment:
Another question - it appears that we are adding a thermo ddt?  Is that correct and if so what are its elements?